README.md

```diff
@@ -1,2 +1,83 @@
 # Robot-Bug
 Srishti 2020
+## Abstract:<br/>
+<p align="justify">
+Aim of the project is to make a robot bug that can be used to collection information from places that are inaccessible for humans<br/>
+
+![image](https://user-images.githubusercontent.com/55125159/88085295-47a00a80-cba3-11ea-95ea-f0bd5559e5a1.jpg)
+
+ 
+## Motivation:<br/>
+<p align="justify">
+We have been hearing of robots being used mainly in factories for automating time-consuming processes and how they have radically changed the way work is being done. In the current times, the focus is on how the use of robotics outside the factory can change our lives over the coming decades. This is not just limited to easing out the day to day tasks but also includes the use of robots for more complex tasks. 
+
+## Mechanical Aspects of Design:
+<p align="justify">
+1)	3 servo motors have been used for the project. The function of the servo motor placed along the side face of bug’s body rotates the bevel gears which is turn raise the arm up. <br/>
+2)	Once the arms are raised, the 2 servo motors being attached on the arms(1 on each) then moves the arms forward. <br/>
+3)	Hence first the arms are raised, then they move forward, touch down and then bot moves forward. <br/>
+4)	A castor wheel is also attached so that the bot can move smoothly without any restrictions due to friction. <br/> 
+
+
+## Electronic Aspects of Design:
+<p align="justify">
+1)	Arduino has been attached to the body which controls the servo motors’ performance like angle of rotation and speed etc.<br/>
+2)	The speed and the angles have been optimized as per the design and weight of the bug.<br/>
+3)	A rechargeable 11V battery has also been connected to Arduino for power supply.<br/>
+
+## Flowchart
+
+![Screenshot (20)](https://user-images.githubusercontent.com/55125159/88693423-e46f2480-d11c-11ea-87b7-0cd2659561b9.png)
+
+
+## Cost Structure:
+|Components    |Number Used|Cost(INR.) |
+|---           |---	   |---        |
+|Arduino UNO   |1          |500        |
+|Battery       |1          |400        |
+|Caster Wheel  |1          |25         |
+|Servo Motors|3 |250  |
+|Bluetooth Module |1   |350   |
+|Miscellaneous   |---|150 |
+|Total |2175|
+	
+                                                            
+
+## Application:
+<p align="justify">
+1)	Making surveillance of any disaster-affected area where human beings can’t go.<br/>
```

Comment:
In lists like Applications, Limitations, Future Improvements, use `<ol><li>Line1<\li><li>Line2<\li>...<\ol>` as it is coming under HTML tag.